src/DataAccess/Repositories/EarningsQueryRepository.cs

```diff
@@ -30,7 +30,8 @@ public async Task<ProviderEarningsSummary> GetProviderSummary(long ukprn, short
             {
                 TotalLevyEarningsForCurrentAcademicYear = await DbContext.Earning.Where(x => x.UKPRN == ukprn && x.AcademicYear == academicYear && x.FundingType == FundingType.Levy).SumAsync(x => x.Amount),
                 TotalNonLevyEarningsForCurrentAcademicYear = await DbContext.Earning.Where(x => x.UKPRN == ukprn && x.AcademicYear == academicYear && x.FundingType == FundingType.NonLevy).SumAsync(x => x.Amount),
-            };
+                Ukprn =  DbContext.Earning.Where(x => x.UKPRN == ukprn && x.AcademicYear == academicYear).Select(x => x.UKPRN).FirstOrDefault(),
```

Comment:
This doesn't make sense, why not just use the value passed in?  Although given it's an input, it doesn't need to be an output.